reconcile/utils/mr/base.py

```diff
@@ -52,7 +52,7 @@ def __init__(self) -> None:
         # of the child class.
         self.sqs_msg_data = {**self.__dict__}
 
-        self.labels: Iterable[str] = [DO_NOT_MERGE_HOLD]
+        self.labels: list[str] = [DO_NOT_MERGE_HOLD]
```

Comment:
Ah, it's the base class `MergeRequestBase`. Please revert this commit. `Iterable` would be better